tests/integration/release/test_release.py

```diff
@@ -198,6 +198,10 @@ def unnoise_data(data_: pd.DataFrame) -> pd.DataFrame:
             data_[get_noised_columns(data_)] = data_[get_prenoised_columns(data_)]
             return data_
 
+        def remove_missing_rows(data_: pd.DataFrame) -> pd.DataFrame:
+            """Removes rows that are missing in all columns."""
+            return data_[~get_missingness_data(data_).all(axis=1)]
```

Comment:
I don't think this is right? Omit row doesn't affect missingness - it removes the row entirely. I believe we want:
```python
def remove_missing_rows(data_: pd.DataFrame) -> pd.DataFrame:
    """Removes rows that are missing in all columns."""
    return data_[Dataset.is_missing(get_noised_data(data_)).all(axis=1)]
```